ofl/frankruhllibre/debug.log

```diff
@@ -0,0 +1,2 @@
+[1016/171748.853:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
again these log files. What's the purpose of these? It seems to me that they should not be included in the repo.